frappe/email/doctype/newsletter/newsletter.py

```diff
@@ -72,7 +72,7 @@ def queue_all(self):
 			files = frappe.get_all("File", fields = ["name"], filters = {"attached_to_doctype": "Newsletter",
 				"attached_to_name":self.name}, order_by="creation desc")
 
-			for a in files:
+			for file in files:
```

Comment:
@manassolanki,

Also make the changes in `frappe.throw` statement replace the `a` by `file`